app/celery/nightly_tasks.py

```diff
@@ -394,3 +395,13 @@ def delete_unneeded_notification_history_for_specific_hour(start_datetime: str,
     )
 
     delete_notification_history_between_two_datetimes(start_datetime, end_datetime)
+
+
+@notify_celery.task(name="update-report-status-to-deleted")
+def update_report_status_to_deleted():
+    try:
+        update_report_requests_status_to_deleted()
+        current_app.logger.info("Successfully updated report status to deleted.")
+    except SQLAlchemyError as e:
```

Comment:
Optional: without this error handling we'd still see the error and stack trace in the logs and know to investigate, so we can get away without catching and then re-raising to add an additional log line